drivers/espi/host_subs_npcx.c

```diff
@@ -148,6 +149,18 @@ struct host_sub_npcx_data {
 	uint8_t plt_rst_asserted; /* current PLT_RST# status */
 	uint8_t espi_rst_asserted; /* current ESPI_RST# status */
 	const struct device *host_bus_dev; /* device for eSPI/LPC bus */
+#ifdef CONFIG_ESPI_PERIPHERAL_DEBUG_PORT_80
+	struct ring_buf port80_ring_buf;
+	uint8_t port80_data[CONFIG_ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_RING_BUF_SIZE];
+	struct k_work work;
+#endif
+};
+
+struct npcx_dp80_buf {
+	union {
+		uint16_t u16;
+		uint8_t u8[2];
```

Comment:
Pick meaningful names for the fields. Browsing the code I see `struct can_frame` using `data` and `data_32` for a similar case, something like that would work.

Reply:
done.